Completion/Linux/Command/_nmcli

```diff
@@ -1,244 +1,369 @@
-#compdef nmcli
+#compdef _nmcli nmcli
 
-local context state line expl
-typeset -A opt_args
+# compatible with nmcli 1.8.4
 
-if [[ -z $_nmcli_version ]]; then
-    _nmcli_version="${"$(_call_program nmcli $words[1] --version)"##*version }"
-fi
+_nmcli() {
+    local line    _arguments -C \
+        {'(-terse)-t','(-t)-terse'}"[terse output]" \
+        {'(-pretty)-p','(-p)-pretty'}"[pretty output]" \
+        {'(-mode)-m','(-m)-mode'}"[output mode]: :(tabular multiline)" \
+        {'(-colors)-c','(-c)-colors'}"[whether to use colors in output]: :(auto yes no)" \
```

Comment:
Per-match descriptions are missing. The first should be "mode", the second could be just "value" if you can't think of anything better.